crates/pg_configuration/src/migrations.rs

```diff
@@ -0,0 +1,17 @@
+use biome_deserialize_macros::{Merge, Partial};
+use bpaf::Bpaf;
+use serde::{Deserialize, Serialize};
+
+/// The configuration of the filesystem
+#[derive(Clone, Debug, Deserialize, Eq, Partial, PartialEq, Serialize, Default)]
+#[partial(derive(Bpaf, Clone, Eq, PartialEq, Merge))]
+#[partial(serde(rename_all = "snake_case", default, deny_unknown_fields))]
+pub struct MigrationsConfiguration {
+    /// The directory where the migration files are stored
+    #[partial(bpaf(long("migrations-dir")))]
+    pub migrations_dir: String,
```

Comment:
I don't know if we use [bpaf-derive](https://crates.io/crates/bpaf_derive/0.1.0), but if we do, we *could* also parse into a `PathBuf`. 

Examples of that were hard to find, so maybe that's not the recommended way – I'm not sure

Reply:
that would be preferred, but `biome`s `Merge` does not work with `PathBuf` :(